CHANGELOG.md

```diff
@@ -10,11 +10,28 @@
 
 ### 📈 Features/Enhancements
 
+- Update ouiTextSubduedColor in `next` dark theme ([#973](https://github.com/opensearch-project/oui/pull/973))
+
+### 🐛 Bug Fixes
+
+### 🚞 Infrastructure
+
+### 📝 Documentation
+
+### 🛠 Maintenance
+
+### 🪛 Refactoring
+
+### 🔩 Tests
+
+## [`1.3.0`](https://github.com/opensearch-project/oui/tree/1.3.0)
```

Comment:
It looks like we're also missing the sections for other releases:

- `1.2.0`
- `1.1.2`